unstructured/partition/text.py

```diff
@@ -40,22 +41,19 @@ def partition_text(
         The string representation of the .txt document.
     """
 
-    if not any([filename, file, text]):
-        raise ValueError("One of filename, file, or text must be specified.")
+    # Verify that only one of the arguments was provided
+    exactly_one([filename, file, text], ["filename", "file", "text"])
 
-    if filename is not None and not file and not text:
+    if filename:
```

Comment:
```suggestion
    if filename is not None:
```
I think it's better to be explicit anytime you want to guard against `None`, so I'm tempted to flag it on every line, but it doesn't always cause problems with functionality.

Here, however, if `filename == ""` and `file` and `text` are `None`, it will bypass all the clauses in the `if` statement and `file_text` will be undefined in line 57. I think it'd be better to get a `FileNotFoundError` in line 48 trying to open a file named `""`.

Reply:
I'll reintroduce `is not None`.